tools/crm_mon_print.c

```diff
@@ -54,6 +54,48 @@ static int print_node_attributes(pcmk__output_t *out, pe_working_set_t *data_set
 static int print_failed_actions(pcmk__output_t *out, pe_working_set_t *data_set,
                                 gboolean print_spacer);
 
+static gboolean
+str_in_unames(gchar **unames, const gchar *s) {
+    return unames && g_strv_contains((const gchar * const*) unames, s);
+}
```

Comment:
An irrelevant aside ... pacemaker's use of "uname" is inaccurate. Node names don't have to match system unames. But we need to keep it for consistency now :)